From: Roman Lebedev <lebedev.ri@gmail.com>
Date: Tue, 28 Sep 2021 11:36:38 +0300
Subject: CMake: install docs into halide subdirectory of doc dir (#6267)

Forwarded: https://github.com/halide/Halide/pull/6267
---
 packaging/CMakeLists.txt | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/packaging/CMakeLists.txt b/packaging/CMakeLists.txt
index c032a83..c8d4334 100644
--- a/packaging/CMakeLists.txt
+++ b/packaging/CMakeLists.txt
@@ -99,7 +99,7 @@ install(FILES
         ${Halide_SOURCE_DIR}/README_rungen.md
         ${Halide_SOURCE_DIR}/README_webassembly.md
         COMPONENT Halide_Documentation
-        TYPE DATA)
+        TYPE DOC)
 
 ##
 # Tools
@@ -130,7 +130,7 @@ install(FILES ${Halide_SOURCE_DIR}/src/autoschedulers/adams2019/autotune_loop.sh
 
 if (WITH_TUTORIALS)
     install(DIRECTORY ${Halide_SOURCE_DIR}/tutorial
-            TYPE DATA
+            TYPE DOC
             COMPONENT Halide_Documentation
             FILES_MATCHING
             PATTERN "*.cpp"
@@ -209,7 +209,7 @@ install(FILES
 ##
 
 if (WITH_DOCS)
-    install(DIRECTORY ${Halide_BINARY_DIR}/doc/html/
+    install(DIRECTORY ${Halide_BINARY_DIR}/doc/html
             TYPE DOC
             COMPONENT Halide_Documentation)
 endif ()
