From: Zalman Stern <zalman@google.com>
Date: Wed, 7 Jul 2021 16:31:06 -0700
Subject: Track argument change to LLVM's CreateMaskedLoad. (#6130)

Forwarded: https://github.com/halide/Halide/pull/6130
---
 src/CodeGen_LLVM.cpp | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/CodeGen_LLVM.cpp b/src/CodeGen_LLVM.cpp
index 511ffb3..879a2d5 100644
--- a/src/CodeGen_LLVM.cpp
+++ b/src/CodeGen_LLVM.cpp
@@ -2317,7 +2317,11 @@ llvm::Value *CodeGen_LLVM::codegen_dense_vector_load(const Type &type, const std
         Instruction *load_inst;
         if (vpred != nullptr) {
             Value *slice_mask = slice_vector(vpred, i, slice_lanes);
+#if LLVM_VERSION >= 130
+            load_inst = builder->CreateMaskedLoad(slice_type, vec_ptr, llvm::Align(align_bytes), slice_mask);
+#else
             load_inst = builder->CreateMaskedLoad(vec_ptr, llvm::Align(align_bytes), slice_mask);
+#endif
         } else {
             load_inst = builder->CreateAlignedLoad(vec_ptr->getType()->getPointerElementType(), vec_ptr, llvm::Align(align_bytes));
         }
