From: Roman Lebedev <lebedev.ri@gmail.com>
Date: Thu, 6 Jan 2022 15:11:59 +0300
Subject: Fixup libhalide version/soversion for debian package

Forwarded: not-needed
---
 src/CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 390fee9..94cff31 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -542,8 +542,8 @@ mark_as_advanced(Halide_SOVERSION_OVERRIDE)
 
 set_target_properties(Halide PROPERTIES
                       POSITION_INDEPENDENT_CODE ON
-                      VERSION ${Halide_VERSION}
-                      SOVERSION ${Halide_SOVERSION_OVERRIDE})
+                      SOVERSION ${Halide_SOVERSION_OVERRIDE}
+                      OUTPUT_NAME Halide${Halide_VERSION_MAJOR})
 
 # Note that we (deliberately) redeclare these versions here, even though the macros
 # with identical versions are expected to be defined in source; this allows us to
